### DISCUSSION TITLE
Grbl error does not execute commands R

### DISCUSSION THREAD

**Discussion by elshad66** [arkypita/LaserGRBL#1536] [Q&A]
Hello, as I understand the grbl supports R commands, then why doesn't it, I use a macro for corel to 
generate the code, it gives the code correctly, 
 I use the firmware from the laser grbl for only x y 02-v1.1h-custom, XY Homing-20190830.hex 
the same circles if  copy and paste, then the grbl one cuts the other does not cut, each time in different ways,
 the same code send the first time does not cut the second or third time to cut, or vice versa

G0 Z0.0000  
N10 ( NEXT SHAPE ) 
N20 G00 X34.8831 Y-4.3512 Z0.0000 
N30 G01 Z-0.1000 F2000.0 
N40 G03 X36.8831 Y-4.3512 R1.0000 F2000.0000 
N50 G03 X34.8831 Y-4.3512 R1.0000 
N60 G00 Z0.0000 
N70 ( NEXT SHAPE ) 
N80 G00 X34.8331 Y-8.3512 Z0.0000 
N90 G01 Z-0.1000 F2000.0 
N100 G02 X36.9331 Y-8.3512 R1.0500 F2000.0000      INVALID  GCODE  ID:34
N110 G02 X34.8331 Y-8.3512 R1.0500                           INVALID  GCODE  ID:33
N120 G00 Z0.0000 
N130 ( NEXT SHAPE ) 
N140 G00 X34.7831 Y-12.3512 Z0.0000 
N150 G01 Z-0.1000 F2000.0 
N160 G03 X36.9831 Y-12.3512 R1.1000 F2000.0000      INVALID  GCODE  ID:34
N170 G03 X34.7831 Y-12.3512 R1.1000                           INVALID  GCODE  ID:33
N180 G00 Z0.0000 
N190 ( NEXT SHAPE ) 
N200 G00 X34.5368 Y-16.4686 Z0.0000 
N210 G01 Z-0.1000 F2000.0 
N220 G02 X36.8368 Y-16.4686 R1.1500 F2000.0000 
N230 G02 X34.5368 Y-16.4686 R1.1500 
N240 G00 Z0.0000

**Comment by elshad66** [comment]
Added a screen
http://moemesto.ru/elshad66/file/15181688/display/2021-07-12.jpg

**Comment by arkypita** [comment]
First: this is not a LaserGRBL issue 
Second: grbl does not support R command

![Screenshot_20210712_205810_com android chrome](https://user-images.githubusercontent.com/8782035/125341609-168c8d00-e354-11eb-9fa2-1e390c5bba99.jpg)
